test/functorch/test_ops.py

```diff
@@ -36,6 +36,11 @@
     is_valid_inplace_sample_input,
     loop2,
 )
+from torch.testing._internal.autograd_function_db import (
+    autograd_function_db
+)
+
+torch._C._set_autograd_function_extension_enabled(True)
```

Comment:
Would it be better if we did this in a context manager or something? Like if someone runs this file then goes to run other stuff, this would unexpectedly affect the run of those programs?

Reply:
Sure. It doesn't matter most of the time, but it can matter.

run_test.py executes the tests by shelling out for each one, so there is no problem for that (the subprocess has its own pytorch). If a user ran `python test/functorch/test_ops.py` followed by `python test/test_autograd.py`, both of these happen in different processes, so that's also OK.

Now the fun starts when someone uses pytest over a folder, like `pytest test/` or multiple files. I don't know what happens here so I'll just use the context manager version of the flag and call it a day

Reply:
Interesting, I didn't think of pytest. Another situation is if someone imports something from this file?